docs/imviz/displayimages.rst

```diff
@@ -0,0 +1,66 @@
+.. _imviz-display-images:
+
+*****************
+Displaying Images
+*****************
+
+Imviz uses image viewers to visualize data from compatible files.
+
+Selecting Data Set
+==================
+
+.. seealso::
+
+    :ref:`Selecting Data Set<cubeviz-selecting-data>`
+        Documentation on selecting data sets in the jdaviz viewers.
+
+
+
+Adding New Viewers
+==================
+
+In the toolbar towards the top of the UI, there is a :guilabel:`picture with +` icon. By
+clicking this icon, you can add new viewers to the application. You can then select from the data
+that has been loaded into the application to be visualized in these extra viewers. You can then utilize some of the
+Imviz-specific features, like Linked Pan/Zoom.
+
+Defining Spatial Regions
+========================
+
+Spatial regions can be created the same way as in Cubeviz.
+
+.. seealso::
+
+    :ref:`Defining Spatial Regions<spatial-regions>`
+        Documentation on defining spatial regions in an image viewer.
+
+Blinking
+========
+
+Blinking is an Imviz-specific functionality that allows a user to quickly switch
+between viewing two images. This can be done by selecting the :guilabel:`hammer and screwdriver` icon
+and then clicking the :guilabel:`right arrow` icon. You can also access blinking
+by selecting the "B" key on your keyboard.
+
+Pan/Zoom and Linked Pan/Zoom
+============================
+
+Linked pan zoom is an Imviz-specific feature that allows the user to Pan and Zoom
+images in multiple different viewers simultaneously. This works by matching images
+based on their WCS (if present) or linking images by pixel.
```

Comment:
This will be true after we merge #735.
```suggestion
images in multiple different viewers simultaneously. This works by matching images
based on the way they are linked together. Images are linked by pixels on load time,
but you can re-link them via WCS using `~jdaviz.configs.imviz.helper.Imviz.link_data`.
```